generated-platform-project/quarkus-qpid-jms/integration-tests/quarkus-qpid-jms-integration-tests/pom.xml

```diff
@@ -25,8 +25,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.quarkus</groupId>
+      <groupId>io.quarkiverse.artemis</groupId>
       <artifactId>quarkus-test-artemis</artifactId>
+      <version>${quarkus-amazon-services.version}</version>
```

Comment:
@aloubyansky I noticed the generation decided to use this property to set the version here, which I found a bit unexpected.

The property value does resolve to the expected "1.0.1" version currently, is that the only reason it did so? I assume it wouldnt if it resolved to something else? Anything I need to do differently?

Reply:
No, it's a coincidence and is safe to ignore.

Reply:
ok great